cybershell/bypass_techniques.py

```
@@ -0,0 +1,620 @@
+            {"X-Method-Override": "GET"},
+        ]
+        
+        for header_bypass in header_bypasses:
+            bypass_headers = headers.copy()
+            bypass_headers.update(header_bypass)
+            request = {
+                "method": method,
+                "path": path,
+                "headers": bypass_headers,
+                "params": params.copy(),
+                "bypass_technique": f"header_{list(header_bypass.keys())[0]}",
```

Ruff (0.12.2)

Prefer `next(iter(header_bypass.keys()))` over single element slice

Replace with `next(iter(header_bypass.keys()))`

(RUF015)